vmdb/app/views/layouts/listnav/_container_group.html.haml

```diff
@@ -35,4 +35,14 @@
           %li
             = link_to("#{ui_lookup(:table => "container_node")}: #{@record.container_node.name}",
                 {:controller => 'container_node', :action => 'show', :id => @record.container_node},
-                :title => _("View %s %s") % [ui_lookup(:table => "container_node"), @record.container_node.name])
\ No newline at end of file
+                :title => _("View %s %s") % [ui_lookup(:table => "container_node"), @record.container_node.name])
+        - if role_allows(:feature => "container_service_show_list")
+          - num_services = @record.number_of(:container_services)
+          - if num_services == 0
+            %li.disabled
+              = link_to(_("%s (0)") % ui_lookup(:table => "container_services"), "#")
```

Comment:
@zeari can you change ui_lookup call here and on line 46 to ui_lookup(:tables => "container_service") to match the label on summary screen.